src/paperless/settings.py

```diff
@@ -566,3 +566,7 @@ def default_threads_per_worker(task_workers):
         d = dateparser.parse(s)
         if d:
             IGNORE_DATES.add(d.date())
+
+ENABLE_UPDATE_CHECK = os.getenv("PAPERLESS_ENABLE_UPDATE_CHECK", "default")
+if ENABLE_UPDATE_CHECK != "default":
+    ENABLE_UPDATE_CHECK = __get_boolean("PAPERLESS_ENABLE_UPDATE_CHECK", "true")
```

Comment:
I understood this that if the value is anything else than `false` or [unset] the feature will be enabled, correct? wouldn´t it be better to do this vice-versa - ony enable it if the value is explicit `true`?

Reply:
No, the default on line 572 is misleading, because an unset `PAPERLESS_ENABLE_UPDATE_CHECK` would already be caught in line 570. Unless the environment changes mid-script, this value is never used. `__get_boolean` itself errs on the side of "false".